### ISSUE TITLE
How to change learning rate when using 'train_from'

### ISSUE THREAD

**Issue by StephenKyung** [OpenNMT/OpenNMT-py#2084]
Hello, I trained model with -learning_rate 0.001 and save the checkpoints
then I want to change it to 0.01 when I train from the check points.

How to change learning rates when train from saved check points?

Change flag of command : 

CUDA_VISIBLE_DEVICES=6 python train.py -data /data/math/WAPL/processed/demo -save_model /data/math/WAPL/model/convmath_decoder/ -save_checkpoint_steps 21200 -valid_steps 21200 -start_decay_steps 99999999 -log_file /data/math/WAPL/log/log_convmathDec.txt -src_vocab /data/math/WAPL/latex_vocab.txt -gpu_ranks 0 -train_from /data/math/WAPL/model/convmath_decoder/_step_275600.pt -learning_rate 0.01

not working.
please help.

**Comment by francoishernandez** [comment]
Check the `reset_optim` option. Else, if you want some more detailed modifications you can always load your checkpoint in a python shell, modify the checkpoint['optim'] or checkpoint["opts'] and save it back.
